submissions/mod-comparison/mod-comparison.md

```diff
@@ -0,0 +1,92 @@
+
+AngelBobs keeps the same basic goal of launching a rocket, however all of the recipes leading up to it are changed significantly, making it much more complicated. For example, in the early game ores need to be crushed before they can be smelted. But then later on, you discover that if you sort them or wash them, you can get multiple different metal ores out, and if you then process those through further steps you can boost the number of plates you produce for each unit of ore to at least double, and then even further if you start to use modules. 
+
+![Iron ore processing](ironoreprocessing.png)
+
+Space Exploration provides a continuation of the base game. When you launch a rocket, you don't get the "congratulations, you won!" message, instead you realise that you've just scratched the surface and there's far more science to be done. The different types of space science all have different feels and all require different resources - resources which you have to find on other planets. The early game is very similar to vanilla, but it gives you a solar system and deep space to explore once you've got rocketry going. 
+
+Industrial Revolution also keeps the goal of launching the rocket, but adds in extra early game content with different eras of systems. You begin with steam power, using it to directly power inserters, assembly machines and so on, until you finally research electricity and can start using more normal machines. It also adds in some extra processing steps for ores and new recipes for circuits, but not to the same extent or complexity as AngelBobs. 
+
+![Steam powered assembly machines](IRsteam.png)
+
+### Difficulty Curve
+
+The difficulty in a vanilla megabase is largely around designing at a scale you haven't worked at before. This can mean trying to work with longer trains, designing a rail system that can cope with them or planning out enormous arrays of machines to produce everything you need in sufficient quantities. Once you've launched your first rocket, however, there aren't any new puzzles to solve, it's all about making your existing solutions run faster and more efficiently. There's also nothing in the game to stop you sprawling further and further, so eventually the only limit to your expansion is in how powerful your computer is and whether it can keep up with all the simulation! This brings up the main challenge in megabase design - making your designs efficient enough that you can squeeze in that extra rocket per minute. It's definitely a challenge, but not one that I've personally been very interested in. I'd rather be solving new puzzles, rather than trying to make my existing solutions more efficient. 
```

Comment:
I get what you mean about megabasing being mainly about optimizing vanilla features. I disagree with the point that there are no new puzzles to solve after launching the rocket: While one side of megabasing is to make what you have more efficient, on the other hand there is the novelty of coming up with entirely new designs specifically because you will be producing things at scale. More than half of the time i spent on my vanilla megabase was on designing blueprints and learning/deciding how to group and connect base areas etc.